components/Square.tsx

```diff
@@ -3,9 +3,13 @@ import{
     StyleSheet,
     Text,
     View,
-    TouchableNativeFeedback
+    ToastAndroid, 
+    Platform, 
+    Alert,
 } from 'react-native';
+import GestureRecognizer from 'react-native-swipe-gestures';
 
+const Tile_Size: number = 35;
```

Comment:
Store this constant as well as the color constants in separate files that store constants and can be imported in. That way you can easily change it later if needed.